propagation.go

```diff
@@ -93,7 +99,7 @@ type binaryPropagator struct {
 	buffers sync.Pool
 }
 
-func newBinaryPropagator(tracer *Tracer) *binaryPropagator {
+func NewBinaryPropagator(tracer *Tracer) *binaryPropagator {
```

Comment:
exported func NewBinaryPropagator returns unexported type *jaeger.binaryPropagator, which can be annoying to use